lib/phoenix_swagger/conn_validator.ex

```diff
@@ -89,7 +89,7 @@ defmodule PhoenixSwagger.ConnValidator do
   defp validate_body_params(path, conn) do
     case Validator.validate(path, conn.body_params) do
       :ok -> :ok
-      {:error, [{error, error_path} | _], _path} -> {:error, error, error_path}
+      {:error, [{_error, _error_path} | _] = errors, path} -> {:error, errors, path}
```

Comment:
looks like both `:error` cases have the same behaviour now.
Each case just returns the result from `Validator.validate`, so the function can simply be:

```elixir
defp validate_body_params(path, conn) do
  Validator.validate(path, conn.body_params)
end
```

Reply:
Definitely. Thank you for commeting! @mbuhot